python/kfserving/README.md

```diff
@@ -1 +1,39 @@
-TODO
\ No newline at end of file
+# KFServing
+
+KFServing is a unit of model serving, and every model type (XGBosst, PyTorch, Tensorflow etc.) extends KFServing. At a highlevel it defines a model, a server and storage mechanism for models.
+
+A model has essentially three functions:
```

Comment:
I'd drop this from the docs. This is just an implementation detail that I don't want to codify yet.

Reply:
Done